source/content/cms-admin.md

```diff
@@ -167,11 +167,7 @@ Perhaps the most powerful way to leverage the capabilities of on-server developm
 
 In order to run WP-CLI or Drush commands on Pantheon's development environments, you'll need use our very own command line tool, Terminus:
 
-1. Install [Terminus](/terminus):
-
-  ```bash{promptUser: user}
-  curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar && php installer.phar install
-  ```
+1. [Install Terminus](/terminus/install)
 
 1. [Generate a Machine Token](https://dashboard.pantheon.io/machine-token/create) from **User Dashboard** > **Account** > **Machine Tokens**, then authenticate Terminus:
```

Comment:
Shouldn't this line be incremented to `2.`?

Reply:
hi @uberhacker !
good catch - the markdown treats these as ordered list items and autoincrements each `1.` 
we use the repeated `1.` so that when we edit or move things around, we don't have to go cleaning up numbers too (which the md would ignore anyway, as 1,2,3,4 would be treated the same as 1,5,2,7)

you can try it in github comments (though, the built-in editor is helpful and 1,2,3s for you):

```
1. item one
1. item two
```

or check out [markdownguide.org](https://www.markdownguide.org/basic-syntax#ordered-lists)

Reply:
Hey @EdwardAngert!  Not sure I understand.  I would think if you inserted a new line, markdown would be smart enough to increment all the lines correctly for you.  But I see the extra line between numbers may be resetting the ordered list.  Would it make sense to not add the extra line to keep the ordered list intact?

Reply:
OK, never mind.  I see how this works.  It's confusing but the rendered output is different than the source.